### ISSUE TITLE
Unable to resolve symbol: get-current-account in this context

### ISSUE THREAD

**Issue by nickstares** [jaydeesimon/dropbox-clj-repl#1]
Following the instructions, I got this error when I tried to run `(get-current-account)` from `lein repl`:

```CompilerException java.lang.RuntimeException: Unable to resolve symbol: get-current-account in this context, compiling:(/private/var/folders/zj/ky755ybs3qn7p87pzyyqnds80000gn/T/form-init1452201725993112537.clj:1:1) ```

this is my `project.clj`:

```
(defproject orgmode-sms-inbox "0.1.0"
  :description "FIXME: write description"
  :url "http://example.com/FIXME"
  :license {:name "Eclipse Public License"
            :url "http://www.eclipse.org/legal/epl-v10.html"}
  :user {:plugins [[cider/cider-nrepl "0.12.0-SNAPSHOT"]
                   [refactor-nrepl "0.3.0-SNAPSHOT"]
                   [lein-environ "1.0.3"]]
         
         :dependencies [[alembic "0.3.2"]
                        [org.clojure/tools.nrepl "0.2.7"]]}

  :dependencies [[org.clojure/clojure "1.9.0"]
                 [http-kit "2.1.18"]
                 [compojure "1.4.0"]
                 [ring/ring-defaults "0.1.5"]
                 [org.clojure/java.jdbc "LATEST"]
                 [org.postgresql/postgresql "LATEST"]                 
                 [com.jaydeesimon/dropbox-repl "0.1.1"]] 

  :jvm-opts ["-Xmx1G"]
  :main orgmode-sms-inbox.core)
```

my profiles.clj:
```
{:dev {:env {:access-token "my_own_access_token"}}} 
```
Your help is much appreciated!

**Comment by jaydeesimon** [comment]
Hi Nick,

What happens if you try `(dropbox-repl.core/get-current-account)` instead? Does that work? I think what may be happening is that when you get into the repl (after invoking `lein repl`), you are not in the `dropbox-repl.core` namespace and therefore it can't find the `get-current-account` function without you qualifying it.

Alternatively, you could try switching to the namespace and then invoking it like:

```
=> (in-ns 'dropbox-repl.core)
=> (get-current-account)
# Prints out some useful stuff about my account
```

Let me know if that works. Happy to help.

-Jeff

**Comment by nickstares** [comment]
Hi Jeff,

I tried both of those steps to no avail. In a fresh REPL, `(find-ns 'dropbox-repl.core)` is `nil`.

Thanks so much again for helping!

**Comment by jaydeesimon** [comment]
Oh I see. I just realized you are depending on the project as a library. When you do `lein repl`, you are probably getting an "Access token is nil" exception from this [line.](https://github.com/jaydeesimon/dropbox-repl/blob/master/src/dropbox_repl/core.clj#L20-L22) Is that true? If that's the case, it's failing to define the rest of the vars in that namespace.

I didn't intend the project to be included as a dependency on other projects but all you really need to do is make sure the access token is defined so that assert doesn't get tripped. There are much better ways to design it than what I did but I use weavejester's environ library to find the access token which means it will try to look for a value for `:access_token` in a well-defined order. See the environ README for more info: https://github.com/weavejester/environ but here's a minimal case project that I hope demonstrates what I'm talking about: https://github.com/jaydeesimon/include-dropbox-repl
